lib/manageiq/loggers/base.rb

```diff
@@ -10,6 +10,19 @@ module Loggers
     class Base < Logger
       MAX_LOG_LINE_LENGTH = 8.kilobytes
 
+      class << self
+        def wrap(loggers)
+          if ActiveSupport.gem_version >= Gem::Version.new("7.1.0")
+            require 'active_support/broadcast_logger'
+            ActiveSupport::BroadcastLogger.new(*loggers)
+          else
+            logger = loggers.shift
+            loggers.each { |wrapped_logger| logger.extend(ActiveSupport::Logger.broadcast(wrapped_logger)) }
+            logger
+          end
+        end
```

Comment:
I'm wondering if we should change this interface, to have a the primary logger be the first param, and the broadcase loggers as the rest.  Then in the instance method you don't need to do the .unshift.

So here, something like

```suggestion
        def wrap(source, *loggers)
          if ActiveSupport.gem_version >= Gem::Version.new("7.1.0")
            require 'active_support/broadcast_logger'
            ActiveSupport::BroadcastLogger.new(loggers.unshift(source))
          else
            loggers.each { |l| source.extend(ActiveSupport::Logger.broadcast(l)) }
            source
          end
        end
```

Reply:
Yeah that works